services/app-api/src/handlers/configuration.ts

```diff
@@ -82,4 +90,125 @@ async function getDBClusterID(secretName: string): Promise<string | Error> {
+    emailParameterStore: EmailParameterStore
+): Promise<Omit<EmailConfiguration, 'stage' | 'baseUrl'> | Error> {
+    // Configuring emails using emailParameterStore
+    // Moving setting these emails down here. We needed to retrieve all emails from parameter store using our
+    // emailParameterStore because serverless does not like array of strings as env variables.
+    // For more context see this ticket https://qmacbis.atlassian.net/browse/MR-2539.
+    const emailSource = await emailParameterStore.getSourceEmail()
+    const devReviewTeamEmails =
+        await emailParameterStore.getDevReviewTeamEmails()
+    const helpDeskEmail = await emailParameterStore.getHelpDeskEmail()
+    const cmsReviewHelpEmailAddress =
+        await emailParameterStore.getCmsReviewHelpEmail()
+    const cmsRateHelpEmailAddress =
+        await emailParameterStore.getCmsRateHelpEmail()
+    const oactEmails = await emailParameterStore.getOACTEmails()
+    const dmcpReviewEmails = await emailParameterStore.getDMCPReviewEmails()
+    const dmcpSubmissionEmails =
+        await emailParameterStore.getDMCPSubmissionEmails()
+    const dmcoEmails = await emailParameterStore.getDMCOEmails()
+
+    if (emailSource instanceof Error) return new Error(emailSource.message)
+
+    if (devReviewTeamEmails instanceof Error)
+        return new Error(devReviewTeamEmails.message)
+
+    if (helpDeskEmail instanceof Error) return new Error(helpDeskEmail.message)
+
+    if (cmsReviewHelpEmailAddress instanceof Error)
+        return new Error(cmsReviewHelpEmailAddress.message)
+
+    if (cmsRateHelpEmailAddress instanceof Error)
+        return new Error(cmsRateHelpEmailAddress.message)
+
+    if (oactEmails instanceof Error) return new Error(oactEmails.message)
+
+    if (dmcpReviewEmails instanceof Error)
+        return new Error(dmcpReviewEmails.message)
+
+    if (dmcpSubmissionEmails instanceof Error)
+        return new Error(dmcpSubmissionEmails.message)
+
+    if (dmcoEmails instanceof Error) return new Error(dmcoEmails.message)
+
+    return {
+        emailSource,
+        devReviewTeamEmails,
+        helpDeskEmail,
+        cmsReviewHelpEmailAddress,
+        cmsRateHelpEmailAddress,
+        oactEmails,
+        dmcpReviewEmails,
+        dmcpSubmissionEmails,
+        dmcoEmails,
+    }
+}
+
+// Note: Email settings won't update dynamically if the database changes while this lambda is running.
+// Consider configuring the emailer later to fetch settings on demand.
+async function configureEmailer({
+    emailParameterStore,
+    store,
+    ldService,
+    stageName,
+    emailerMode,
+    applicationEndpoint,
+}: {
+    emailParameterStore: EmailParameterStore
+    store: Store
+    ldService: LDService
+    stageName: string
+    emailerMode: string
+    applicationEndpoint: string
+}): Promise<Emailer | Error> {
+    const removeParameterStore = await ldService.getFeatureFlag({
+        key: 'throwaway-key-email-configuration',
```

Comment:
Is this the correct key?

Reply:
The key in the LD key application should be the user id, which there is none at this point. Whatever value is here is not going to be like our other keys in us in the application. Gave it weird name since this is just getting overidden. We are passing `anonymous` in as well (though not convinced thats working, need to test on next PR).